src/jaguar.toit

```diff
@@ -20,7 +20,7 @@ import .network
 import .uart
 
 interface Endpoint:
-  run device/Device -> none
+  run device/Device [--validate-firmware] -> none
```

Comment:
Why is this done through a callback? Why not just import the jaguar.toit file and call validate-firmware? What am I missing?

Reply:
I hoped I would forget less easily if given as argument.
Changed to a static call.
done.